regions/io/read_ds9.py

```diff
@@ -357,6 +385,19 @@ def line_parser(line, coordsys=None):
                 language_spec[region_type] = itertools.chain((coordinate, coordinate), itertools.cycle((radius,)))
 
             return region_type, parsed_return, parsed_meta, composite, include
+    else:
+        # it is not clear when it is OK to warn
```

Comment:
Can we resolve / improve the discussion on when to warn in this PR, and then change the comment?
It currently reads as an open issue "it is not clear ..."

Maybe just put whatever behaviour you like best, and establish it via tests?